server/package.json

```diff
@@ -19,6 +19,7 @@
   },
   "devDependencies": {
     "@types/async-retry": "^1.4.3",
+    "@types/ffi-napi": "^4.0.5",
```

Comment:
Why do we need it?

Reply:
It is temporary solution. When we fix issue with max features I will prepare new npm version with `@types/ffi-napi` in dependencies.

Reply:
Now it is deleted.